airbyte-server/src/main/java/io/airbyte/server/handlers/DestinationDefinitionsHandler.java

```diff
@@ -131,7 +136,8 @@ private static DestinationDefinitionReadList toDestinationDefinitionReadList(fin
   }
 
   public DestinationDefinitionReadList listLatestDestinationDefinitions() {
-    return toDestinationDefinitionReadList(getLatestDestinations());
+    final DestinationDefinitionReadList output = toDestinationDefinitionReadList(getLatestDestinations());
+    return output;
```

Comment:
I don't think I see a difference here?

Reply:
Sorry, this was a leftover from a debugging attempt. I reverted back to an inline declaration.